developer-docs-site/docs/tutorials/first-transaction-sdk.md

```diff
@@ -0,0 +1,209 @@
+
+In Aptos, each account must have an on-chain representation in order to support receive tokens, coins, and interacting in other DApps. In Aptos, an account represents a medium for storing assets, hence it must be explicitly created. This example leverages the Faucet to create and fund Alice's account and to just create Bob's:
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```
+faucet_client.fund_account(alice.address(), 20_000)
+faucet_client.fund_account(bob.address(), 0)
+```
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+  </TabItem>
+</Tabs>
+
+### Step 4.4: Reading balances
+
+In this step, the SDK translates a single call into the process of querying a resource and reading a field from that resource.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```
+print(f"Alice: {rest_client.account_balance(alice.address())}")
+print(f"Bob: {rest_client.account_balance(bob.address())}")
+```
+
+Behind the scenes, the SDK queries the CoinStore resource for the AptosCoin and reads the current stored value:
+```
+def account_balance(self, account_address: str) -> int:
+    """Returns the test coin balance associated with the account"""
+    return self.account_resource(
+        account_address, "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>"
+    )["data"]["coin"]["value"]
+```
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+  </TabItem>
+</Tabs>
+
+### Step 4.5: Transferring
+
+Like the previous step, this is another helper step that constructs a transaction that transfers the coins from Alice to Bob. For correctly generated transactions, the API will return a transaction hash that can be used in the ensuing step to check on the transaction status. Aptos does perform a handful of validation checks on submission and if any of those fail, the user will instead be given an error. These validations include the transaction signature, unused sequence number, and submitting the transaction to the appropriate chain.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```
+txn_hash = rest_client.bcs_transfer(alice, bob.address(), 1_000)
+```
+
+Behind the scenes, the SDK generates, signs, and submits a transaction:
+```
+def bcs_transfer(
+    self, sender: Account, recipient: AccountAddress, amount: int
+) -> str:
+    transaction_arguments = [
+        TransactionArgument(recipient, Serializer.struct),
+        TransactionArgument(amount, Serializer.u64),
+    ]
+
+    payload = ScriptFunction.natural(
+        "0x1::coin",
+        "transfer",
+        [TypeTag(StructTag.from_str("0x1::aptos_coin::AptosCoin"))],
+        transaction_arguments,
+    )
+
+    signed_transaction = self.create_single_signer_bcs_transaction(
+        sender, TransactionPayload(payload)
+    )
+    return self.submit_bcs_transaction(signed_transaction)
+```
+
+Breaking the above down into pieces:<br/>
+(1) `transfer` internally is a `ScriptFunction` or an entry function in Move that is directly callable.<br/>
```

Comment:
Lol we didn't change the name 'script function'?

Reply:
If someone has time this week, we could change this in types/src/transactions/mod.rs and other areas.. sigh...